test/component/turen/lifetime.test.js

```diff
@@ -77,7 +77,7 @@ test(`scene app should be resumed if cut app exited on middle of asr processing`
   var cutApp = createMockApp(runtime, '1')
   var sceneApp = createMockApp(runtime, '2')
 
-  mock.mockReturns(runtime.component.custodian, 'isPrepared', true)
+  mock.mockReturns(runtime.component.custodian, 'isLoggedIn', true)
```

Comment:
Why this PR change the behavior?

Reply:
The machine state of custodian has been rewrite, and there's no isPrepared at present.

Reply:
+1 on questioning why this API would be changed.

I'd think these two are slight different on if network is available.